calculating_n50_assemblies.py

```diff
@@ -0,0 +1,91 @@
+import glob
+import re
+import gzip
+import operator
+import sys
+import os
+
+def main(in_dir = None, out_dir = None):
+    if os.path.exists(in_dir) == False:
+        sys.exit("Input directory or path incorrect. Please check")
+
+    if os.path.exists(out_dir) == False:
+        os.makedirs(out_dir)
+    
+    file1 = open(os.path.join(out_dir+'/summary_concise.txt'),'w')
+    file2 = open(os.path.join(out_dir+'/summary_verbose.txt'),'w')
+
+    #intializing assembly count
+    assembly_count = 0
+
+    #iterating through each assembly file
+    for file in glob.glob(os.path.join(in_dir+'/*.gz')):
```

Comment:
If a user supplies a directory with non fasta gzipped files your program will error. There should be a check to ensure that you only processes valid fasta files. Refer to google for all valid fasta file extensions.

Reply:
Correct. I did somethings like including adding a list of fasta identifiers and more loops and approval check.